src/extensions/replay/sessionrecording.ts

```diff
@@ -1090,6 +1144,36 @@ export class SessionRecording {
         })
     }
 
+    private _checkUrlTrigger() {
+        if (typeof window === 'undefined' || !window.location.href) {
+            return
+        }
+
+        const url = window.location.href
+
+        if (
+            this._urlTriggers.some((trigger) => {
+                switch (trigger.matching) {
+                    case 'regex':
```

Comment:
Are we okay with just regex to start?

I looked at the ActionMatcher from surveys, but it seems it's doing a lot, we don't need yet. Would you prefer to integrate with that anyway, or is this fine for now?

Reply:
yeah, this is totally fine for now! 

way better to ship a little useful bit than delay while we figure out more complex behaviour! 💪